src/main/java/net/william278/velocitab/commands/VelocitabCommand.java

```diff
@@ -30,38 +30,35 @@ public VelocitabCommand(Velocitab plugin) {
                         AboutMenu.Link.of("https://modrinth.com/plugin/velocitab").withText("Modrinth").withIcon("X").withColor("#589143"));
     }
 
-    @Override
-    public void execute(Invocation invocation) {
-        if (invocation.arguments().length >= 1) {
-             if (invocation.arguments()[0].equalsIgnoreCase("reload")) {
-                 reloadSettings(invocation.source());
-                 return;
-             }
-        }
-
-        sendAboutInfo(invocation.source());
-    }
+    public BrigadierCommand command() {
+        final LiteralArgumentBuilder<CommandSource> builder
+                = LiteralArgumentBuilder.<CommandSource>literal("velocitab")
```

Comment:
ooh, the rare parameterized static method call!
```suggestion
        final LiteralArgumentBuilder<CommandSource> builder = LiteralArgumentBuilder
                .<CommandSource>literal("velocitab")
```
minor nit: newline on the static builder method call rather than before the assignment (=)